procrustes/kopt.py

```diff
@@ -35,7 +35,7 @@
 
 
 def kopt_heuristic_single(fun, p0, k=3, tol=1.0e-8):
-    r"""Find a locally-optimal permutation matrix using the k-opt (greedy) heuristic.
+    r"""Find a locally-optimal one-sided permutation matrix using the k-opt (greedy) heuristic.
```

Comment:
Actually, the two-sided variant should be updated, so I did that and removed the `one-sided` from here because we have a general implementation of `kopt` now that can be applied to any function `f(P)`.